src/main/java/dinkplugin/domain/LootCriteria.java

```diff
@@ -0,0 +1,8 @@
+package dinkplugin.domain;
+
+public enum LootCriteria {
+    ALLOWLIST,
+    DENYLIST,
+    VALUE,
+    RARITY
```

Comment:
should these be renamed to `VALUABLE`/RARE` ?`

Reply:
nah I think this form is good